pkg/util/testing/metrics/metrics.go

```diff
@@ -27,8 +27,9 @@ import (
 )
 
 type GaugeDataPoint struct {
-	Labels map[string]string
-	Value  float64
+	Labels       map[string]string
+	GaugeValue   float64
+	CounterValue float64
```

Comment:
This type is called `GaugeDataPoint`, yet it has two types of values.

Maybe it should be renamed to just `DataPoint` instead?

Reply:
And do we really need the two separate fields?

Reply:
Looking at the code this looks addressed now. @lowang-bh please resolve if you can confirm.